src/main/java/com/rabbitmq/http/client/Client.java

```diff
@@ -1261,6 +1261,29 @@ public void clearMaxQueuesLimit(String vhost) {
     this.deleteIgnoring404(uri);
   }
 
+  public MqttVhostPortInfo getMqttVhostPorts(){
```

Comment:
Can we please rename this to `getMqttPortToVhostMapping`? "MQTT vhost ports" does not make much sense because MQTT does not have a concept of a virtual host.

Reply:
Resolved in latest commit.